pkg/partition/partition_test.go

```diff
@@ -0,0 +1,128 @@
+/*
+Copyright 2020 The OpenEBS Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package partition
+
+import (
+	"github.com/diskfs/go-diskfs/partition/gpt"
```

Comment:
go mod doesn't seems to be having any 'version'.. is it better to use particular tag or version of go-diskfs?

Reply:
Will fix to a commit revision, when we move to go mod soon.